LoopKitUI/Views/Settings Editors/BasalRateScheduleEditor.swift

```diff
@@ -87,18 +110,12 @@ public struct BasalRateScheduleEditor: View {
 
                 }
             }
-        )
-    }
-
-    private var description: Text {
-        Text("Your basal rate of insulin is the number of units per hour that you want to use to cover your background insulin needs.", comment: "Basal rate setting description")
-    }
-
-    private var confirmationAlertContent: AlertContent {
-        AlertContent(
-            title: Text("Save Basal Rates?", comment: "Alert title for confirming basal rates outside the recommended range"),
-            message: Text("One or more of the values you have entered are outside of what Tidepool generally recommends.", comment: "Alert message for confirming basal rates outside the recommended range")
-        )
+        case .flow:
+            return .synchronous { quantitySchedule in
+                let schedule = BasalRateSchedule(dailyItems: quantitySchedule.items, timeZone: .current)!
```

Comment:
The basal schedule should be in the time zone of the pump.. though I understand we don't actually have a configured pump at onboarding time. We'll have to make sure the basal schedule takes on the appropriate time zone once synced to the pump.

In the meantime, I think you'll want `.currentFixed`, rather than `.current`, here.

(Probably worth pinging Pete on this one once he's back.)

Reply:
I'll add that as a todo